canonical-websites/www.ubuntu.com
# Fixes for 18.10

## Done
Review fixes for 18.10 release

## QA
- Check out this feature branch
- Run the site using the command `./run serve`
- View the site locally in your web browser at: [http://0.0.0.0:8001/](http://0.0.0.0:8001/)
- Run through the following [QA steps](https://github.com/canonical-webteam/practices/blob/master/workflow/qa-steps.md)
- @pmahnke check the changes are as expected
